tsconfig.lib.json

```diff
@@ -0,0 +1,11 @@
+{
+	"extends": "./tsconfig",
```

Comment:
I don't understand why we have two TS configs here. Seems like there should only be one.